src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
-
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { request } from '../../api';
+
+export const FindMovie = ({ onAdd }) => {
+  const [movie, setMovie] = useState();
```

Comment:
```suggestion
  const [movie, setMovie] = useState(null);
```
Let it be 'null' initially. It can be handy